packages/berry-builder/sources/build-bundle.js

```diff
@@ -9,6 +9,11 @@ const dynamicLibs = require(`./dynamic-libs`);
 const findPlugins = require(`./find-plugins`);
 const makeConfig = require(`./make-config`);
 
+const pkgJsonVersion = () => {
+  const pkgJson = require("../../../package.json");
```

Comment:
Let's rather use ``require(`${basedir}/package.json`)``. Less hardcoded paths, and the `@berry-build-bundle` user gets to pick the version they want to set in the generated bundle.

Reply:
:+1: